Scaffold docs for Intercom Web and Cloud Actions (#3511)
segmentio/segment-docs: src/connections/destinations/catalog/actions-intercom-web/index.md

---
title: Intercom Web (Actions) Destination
hide-boilerplate: true
hide-dossier: false
hidden: true
id: 62d9daff84a6bf190da9f592
versions:
  - name: 'Intercom Cloud Mode (Actions)'
    link: '/docs/connections/destinations/catalog/actions-intercom-cloud'
  - name: 'Intercom (Classic)'
    link: '/docs/connections/destinations/catalog/intercom'
---

{% include content/plan-grid.md name="actions" %}

> Intercom is a customer communications platform. It shows you who is using your product and makes it easy to personally communicate with your users with targeted content, behavior-driven messages, and conversational support.

Reviewer: ```suggestion
Intercom is a customer communications platform that shows you who is using your product. Intercom also allows you to personally communicate with your users with targeted content, behavior-driven messages, and conversational support.
```

> When you use the Intercom Web (Actions) destination, Segment will load the [Intercom JavaScript library](https://developers.intercom.com/installing-intercom/docs/intercom-for-web){:target="_blank"} for you. The Intercom library enables you to track your user’s events on your website and interact with the Intercom messenger window.

Reviewer: ```suggestion
When you use the Intercom Web (Actions) destination, Segment loads the [Intercom JavaScript library](https://developers.intercom.com/installing-intercom/docs/intercom-for-web){:target="_blank"} for you. The Intercom library enables you to track your user’s events on your website and interact with the Intercom messenger window.
```

> info ""
> The Intercom Web (Actions) destination is in beta and is in active development. Some functionality may change before it becomes generally available.

## Benefits of Intercom Web Mode (Actions) vs Intercom Classic
Intercom Web (Actions) provides the following benefits over the classic Intercom destination:

- **Fewer settings.** Data mapping for actions-based destinations happens during configuration, which eliminates the need for most settings. 
- **Clearer mapping of data.** Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.

> - **Granular control over data sent.** You can customize the conditions under which the events will be sent to Intercom. 

Reviewer: ```suggestion
- **Granular control over data sent.** You can customize the conditions under which the events are sent to Intercom. 
```

> - **Selectively show the Intercom chat widget.**

Reviewer: ```suggestion
- **Selectively shows the Intercom chat widget.**
```

## Getting Started

> 1. From the Segment web app, click **Catalog**.

Reviewer: ```suggestion
1. From the Segment web app, navigate to **Connections > Catalog**.
```
